src/core/model_config.proto

```diff
@@ -623,6 +623,75 @@ message ModelOptimizationPolicy
   PinnedMemoryBuffer output_pinned_memory = 6;
 }
 
+//@@
+//@@.. cpp:var:: message RequestTimeoutPolicy
+//@@
+//@@   Timeout policy for inference requests. If specified, the scheduler will
+//@@   handle pending requests that exceed timeout accordingly.
+//@@
+message RequestTimeoutPolicy
```

Comment:
ModelTimeoutPolicy

Reply:
Maybe we should also make it more general like "ModelQueuePolicy"? If we are going to add max queue size, I think it can be added here.

Reply:
Sounds good. And then have 1 ModelQueuePolicy per priority level (plus a default one).